tox.ini

```diff
@@ -41,7 +41,7 @@ commands =
     python setup.py check -m -r -s -v
 
 deps =
-    docutils
+    docutils==0.12
```

Comment:
UPDATE: `docutils` maintainers do not appear interested in helping without additional information from our end. They are [blaming it on our application](https://sourceforge.net/p/docutils/bugs/302/#7107).